protos/python/vehicle_data_pb2.py

```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: vehicle_data.proto
-# Protobuf Python Version: 4.25.0
```

Comment:
did you generate with some other version of protobuf?

Reply:
yeah, probably need to update my version